### PR TITLE
New package: slurm-wlm 15.08.7

### PR DESCRIPTION
**Pull request by Vaelatern**

### CODE REVIEW

**Comment by Vaelatern** [comment]
Musl compilation is broken -- can't find type `ino_t`.

File: srcpkgs/slurm-wlm/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'slurm-wlm'
+pkgname=slurm-wlm
+version=15.08.7
+revision=1
+wrksrc=slurm-$version
+build_style=gnu-configure
+hostmakedepends="perl cgit python"
+makedepends="munge-devel lua-devel hdf5-devel pam-devel libressl-devel gtk+
+	libnuma-devel ncurses-devel readline-devel rrdtool-devel mysql"
```
Comment (xtraeme) [inline srcpkgs/slurm-wlm/template]: Adding `gtk+` and `mysql` as `makedepends` is useless.

File: srcpkgs/slurm-wlm/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+depends="munge libnuma lua readline ncurses rrdtool libressl pam hdf5 mysql gtk+"
```
Comment (xtraeme) [inline srcpkgs/slurm-wlm/template]: This depends line is completely unnecessary.

File: srcpkgs/slurm-wlm/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'slurm-wlm'
+pkgname=slurm-wlm
+version=15.08.7
+revision=1
+wrksrc=slurm-$version
+build_style=gnu-configure
+hostmakedepends="perl cgit python pkg-config"
```
Comment (xtraeme) [inline srcpkgs/slurm-wlm/template]: Please disable static libs, as you won't need them (no -devel pkg). `configure_args="--disable-static"`.

**Comment by Vaelatern** [comment]
Fixed. In the future it may be useful to re-enable static libs, but currently I can't find anybody who actually builds against slurm-wlm.

**Comment by Vaelatern** [comment]
All issues resolved.

**Pull request merged**